core/store/src/trie/mem/mem_tries.rs

```diff
@@ -52,6 +62,15 @@ impl MemTries {
         }
     }
 
+    pub fn from_frozen_memtries(shard_uid: ShardUId, frozen_memtries: FrozenMemTries) -> Self {
```

Comment:
Can you add a comment? When dealing with resharding it's a good practice to indicate whether shard_uid is that of the parent or the child.